src/components/endpoints/Sign.vue

```diff
@@ -153,7 +154,18 @@ const onSignatureValidation = async () => {
       <div class="field">
         <label class="label">Message</label>
         <div class="control">
-          <textarea v-model="message" class="textarea" rows="3" />
+          <textarea
+            v-if="isSiwe"
+            v-model="siweMessage"
+            class="textarea"
+            rows="3"
+          />
+          <textarea
+            v-if="!isSiwe"
+            v-model="message"
+            class="textarea"
+            rows="3"
+          />
```

Comment:
Why it was extracted into two textareas?

Reply:
Since it's using v-model I wasn't able to dynamically switch between two different models. I wanted to switch the messages with the checkbox on and off.